src/cephadm/cephadm

```diff
@@ -4455,24 +4461,15 @@ def command_run(ctx):
 ##################################
 
 
-def fsid_conf_mismatch(ctx):
-    # type: (CephadmContext) -> bool
-    (config, _) = get_config_and_keyring(ctx)
-    if config:
-        for c in config.split('\n'):
-            if 'fsid = ' in c.strip():
-                if 'fsid = ' + ctx.fsid != c.strip():
-                    return True
-    return False
```

Comment:
this failed, cause there was no daemon on that host, right?

Reply:
this block was refactored into `command_shell` for simplicity.

the failure occurred during `infer_fsid`:
https://github.com/ceph/ceph/blob/a408347e148432c8a8264b0a0022d49f096fc587/src/cephadm/cephadm#L1657-L1668